src/NuGet.Clients/NuGet.CommandLine/Commands/RestoreCommand.cs

```diff
@@ -43,6 +43,9 @@ public class RestoreCommand : DownloadCommandBase
         [Option(typeof(NuGetCommand), "CommandMSBuildPath")]
         public string MSBuildPath { get; set; }
 
+        [Option(typeof(NuGetCommand), "Recursive")]
```

Comment:
Can we go with `-Recurse` instead? It's a common keyword in other tools including PS cmdlets.